ios/ReactTestApp/ContentView.swift

```diff
@@ -53,7 +53,7 @@ final class ContentViewController: UITableViewController {
 
     override public func motionEnded(_: UIEvent.EventSubtype, with event: UIEvent?) {
         guard event?.subtype == .motionShake,
-              let bridge = reactInstance.bridge,
+              let bridge = reactInstance.host?.bridge,
```

Comment:
Any reason not to follow the same structure as lines 196-197?
```
guard let host = reactInstance.host,
      let bridge = host.bridge,
```

Reply:
I normally don't create a separate variable if it's not used outside of the block.